modules/templates/helper.go

```diff
@@ -390,6 +390,42 @@ func NewFuncMap() []template.FuncMap {
 		"Join":        strings.Join,
 		"QueryEscape": url.QueryEscape,
 		"DotEscape":   DotEscape,
+		"Iterate": func(arg interface{}) (items []uint64) {
+			count := uint64(0)
+			switch val := arg.(type) {
+			case uint64:
+				count = val
+			case *uint64:
+				count = *val
+			case int64:
+				count = uint64(val)
+			case *int64:
+				count = uint64(*val)
+			case int:
+				count = uint64(val)
+			case *int:
+				count = uint64(*val)
```

Comment:
This sounds unsafe, not sure which kind of values will ever end up here, but if they are negative values, you will end up with a huge count number.